KabuKit/Classes/core/Swift/Screen.swift

```diff
@@ -26,9 +26,9 @@ public protocol Screen : class {
      - Attention :
      ただし、前のシーンがない場合は戻らず、何も起きない
      */
-    func leave() -> Void
```

Comment:
It is better to that set default argument.
(maybe you thought that too)
But, protocol's function can't be set default argument, 
so I recommend to use overload.
Like this
```Swift
protocol Screen {
  func leave() -> Void
  func leave(_ needsRewind: Bool) -> Void 
}

extension Screen {
  func leave() -> Void {
    leave(true)
  }

  func leave(_ needsRewind: Bool) -> Void {
    ///
```